<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGestionnaire</class>
 <widget class="QMainWindow" name="MainWindowGestionnaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidgetGestionnaire">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPersonnel">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="fenetre">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="boutons">
            <item row="0" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>203</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" rowspan="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonSupprimer">
              <property name="text">
               <string>Del</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" rowspan="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButtonMaj">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>202</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="blocformulaire">
            <item>
             <layout class="QGridLayout" name="formulaire">
              <item row="0" column="0">
               <widget class="QLabel" name="labelNomEmploye">
                <property name="text">
                 <string>Last Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditNom"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelPrenomEmploye">
                <property name="text">
                 <string>First Name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditPrenom"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelSex">
                <property name="text">
                 <string>Sex</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="sexe">
                <item>
                 <widget class="QRadioButton" name="radioButtonMale">
                  <property name="text">
                   <string>Male</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonFemelle">
                  <property name="text">
                   <string>Femelle</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QFormLayout" name="region">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelRegion">
                  <property name="text">
                   <string>Region</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBoxRegion"/>
                </item>
               </layout>
              </item>
              <item row="3" column="1">
               <layout class="QFormLayout" name="departement">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelDepartement">
                  <property name="text">
                   <string>Departement</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBoxDepartement">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QHBoxLayout" name="codepostal">
                <item>
                 <widget class="QLabel" name="labelCodePostal">
                  <property name="text">
                   <string>Postal code</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditCP"/>
                </item>
               </layout>
              </item>
              <item row="4" column="1">
               <layout class="QFormLayout" name="ville">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelVilleEmploye">
                  <property name="text">
                   <string>City</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEditVille"/>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="labelRueEmploye">
                <property name="text">
                 <string>Adress</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEditAdresse"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="labelMail">
                <property name="text">
                 <string>Mail</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="lineEditMail"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelTypeEmploye">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <layout class="QVBoxLayout" name="type">
                <item>
                 <widget class="QRadioButton" name="radioButtonGestionnaire">
                  <property name="text">
                   <string>Gestionnaire</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonControleur">
                  <property name="text">
                   <string>Contr√¥leur</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="labelTelEmploye">
                <property name="text">
                 <string>Phone</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="lineEditTel"/>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="labelDateNaissanceEmploye">
                <property name="text">
                 <string>Birthday</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <layout class="QHBoxLayout" name="anniversaire">
                <item>
                 <widget class="QLineEdit" name="lineEditDN">
                  <property name="inputMask">
                   <string>9999-99-99</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDate">
                  <property name="text">
                   <string>aaaa-mm-jj   </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="labelLoginEmploye">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <layout class="QFormLayout" name="login">
                <item row="0" column="0">
                 <widget class="QLineEdit" name="lineEditLogin"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="labelCaracteresExiges">
                  <property name="font">
                   <font>
                    <family>Monospace</family>
                    <pointsize>9</pointsize>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>min.3 caract√®res </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="validation">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonValider">
                <property name="text">
                 <string>Validate</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVisites">
       <attribute name="title">
        <string>Visit</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabProduits">
       <attribute name="title">
        <string>Products</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabStatistiques">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQUit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQUit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
